Content.Server/GameObjects/Components/Engineering/DisassembleOnActivateComponent.cs

```diff
@@ -13,11 +16,13 @@ public class DisassembleOnActivateComponent : Component
         public override uint? NetID => ContentNetIDs.DISASSEMBLE_ON_ACTIVATE;
 
         [ViewVariables]
-        [DataField("prototype")]
-        public string? Prototype { get; private set; }
+        [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+        public string? Prototype;
```

Comment:
Don't make this a public field... Make this a public property with private setter and use `[field: DataField(blabla...)]` instead.

Reply:
when doing `field:` you can even omit the setter completely